public/javascripts/Gallery/css/gallery.css

```diff
@@ -26,6 +31,14 @@
 
 #horizontal-line {
     border-bottom: 1px solid gray;
+    width: 215px; 
```

Comment:
I think that adding `width: inherit;` to `#card-filter` does something pretty close to this without using a constant

Reply:
This works great!